lib/ts/superTokens.tsx

```diff
@@ -108,52 +103,48 @@ export default class SuperTokens {
         });
     }
 
+    private async initMultitenancyWithDynamicLoginMethods(config: SuperTokensConfig) {
+        const multitenancyRecipe =
+            config.recipeList.find(({ recipeID }) => recipeID === Multitenancy.RECIPE_ID) ?? Multitenancy.init({});
+        const appInfo = normaliseInputAppInfoOrThrowError(config.appInfo);
+        const enableDebugLogs = config.enableDebugLogs === true;
+
+        // initialize auth-react and web-js multitenancy recipes
+        const multitenancyInstance = multitenancyRecipe.authReact(appInfo, enableDebugLogs) as Multitenancy;
+        multitenancyRecipe.webJS(appInfo, config.clientType, enableDebugLogs);
+
+        const tenantID = multitenancyInstance.config.getTenantID?.();
```

Comment:
`?.()` is a bit hard to read; please add a default implementation for this.
Is this prop used in `web-js`, btw?